astropy/nddata/tests/test_ccddata.py

```diff
@@ -279,7 +278,7 @@ def test_setting_uncertainty_with_array(ccd_data):
 
 def test_setting_uncertainty_wrong_shape_raises_error(ccd_data):
     with pytest.raises(ValueError):
-        ccd_data.uncertainty = np.random.random(size=2 * ccd_data.shape)
```

Comment:
Oops, this made a `100,100,100,100` array (which is a bit much for my 2GB laptop)

Reply:
Yeah, I already fixed that downstream (https://github.com/astropy/ccdproc/pull/579) but forgot that the test had been copied over...